shared-libs/transitions/src/lib/infodoc.js

```diff
@@ -82,19 +82,24 @@ const updateInfoDoc = (doc, legacyRev) => {
 };
 
 const updateTransition = (change, transition, ok) => {
-  return findInfoDoc(db.sentinel, change).then(doc => {
-    doc = doc || change.info;
-    doc.transitions = doc.transitions || {};
-    doc.transitions[transition] = {
-      last_rev: change.doc._rev,
-      seq: change.seq,
-      ok: ok,
-    };
-
-    return db.sentinel.put(doc).catch(err => {
-      logger.error('Error updating metaData: %o', err);
+  const info = change.info;
+  info.transitions = info.transitions || {};
+  info.transitions[transition] = {
+    last_rev: change.doc._rev,
+    seq: change.seq,
+    ok: ok,
+  };
+};
+
+const updateTransitions = change => {
```

Comment:
I think this function should be named differently than the one above because it's behaving quite differently. What about `save` or something?

Reply:
Done!